sql_generators/glean_usage/templates/events_stream_v1.query.py

```diff
@@ -77,12 +78,19 @@ def main(submission_date, billing_project, destination_table, temp_dataset, slic
                 ],
             ),
         )
-        print(
+        logging.info(
             f"Writing sample id {min_sample_id} to {max_sample_id} to {temp_table}: {job.path}"
         )
         job.result()
         temp_tables.append(job.destination)
 
+    # update destination table schema if the source has changed
+    temp_table_schema = client.get_table(temp_tables[0]).schema
+    if temp_table_schema != destination_table.schema:
```

Comment:
One other minor concern is if we ever have column descriptions in this table, presumably the temp table won't have those column descriptions, so this method of comparing schemas would detect that as a difference.

However, there currently aren't column descriptions, so we can deal with that issue when the time comes.